README.md

```diff
@@ -217,14 +218,17 @@ While preparing a Virtual Machine that will host SonarQube database, portal and/
 		sonar.jdbc.url=jdbc:jtds:sqlserver://localhost;databaseName=sonar;integratedSecurity=true;”
 		```
 
-6. **Download and install latest SonarQube C# plugin**
+6. <a name="installLatestPluginAnchor"></a>**Download and install latest SonarQube C# plugin**

```

Comment:
This isn't a heading so we have to explicitly define an anchor that we can link to.